### PR TITLE
[INLONG-5084][Manager] Support create Greenplum databases and tables

### LINKED ISSUE #5084: [Feature][Manager] Support create Greenplum databases and tables
### Description

To support the creation of Greenplum databases and tables, you can refer to the class `HiveResourceOperator`.

### Use case

_No response_

### Are you willing to submit PR?

- [ ] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by haibo-duan**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- Fixes #5084 

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve?*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

*(Please pick either of the following options)*

- [ ] This change is a trivial rework/code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a follow-up issue for adding the documentation

### CODE REVIEW

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sink/greenplum/GreenplumJdbcUtils.java [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+     * Check whether the schema exists.
+     *
+     * @param conn JDBC Connection  {@link Connection}
+     * @param schemaName Greenplum schema name
+     * @return true if schema exist in the table, otherwise false
+     * @throws Exception on check column exist error
+     */
+    public static boolean checkSchemaExist(final Connection conn, final String schemaName) throws Exception {
+        boolean result = false;
+        if ("public".equals(schemaName)) {
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sink/greenplum/GreenplumJdbcUtils.java:206]: Suggest extracting the `public` as a constant.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sink/greenplum/GreenplumJdbcUtils.java [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+     *
+     * @param conn JDBC Connection  {@link Connection}
+     * @param schemaName Greenplum schema name
+     * @param tableName Greenplum table name
+     * @param column Greenplum table column name
+     * @return true if column exist in the table, otherwise false
+     * @throws Exception on check column exist error
+     */
+    public static boolean checkColumnExist(final Connection conn, final String schemaName, final String tableName,
+            final String column)
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sink/greenplum/GreenplumJdbcUtils.java:179]: Can be merged into one line.

File: inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/sink/GreenplumSinkServiceTest.java [hunk 3]
```diff
@@ -109,4 +114,65 @@ public void testGetAndUpdate() {
         deleteSink(sinkId);
     }
 
+
+    /**
+     * Just using in local test.
+     */
+    @Test
+    public void testDbResource() {
+        final String url = "jdbc:postgresql://192.168.162.136:5432/testdb";
```
Comment (healchow) [inline inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/sink/GreenplumSinkServiceTest.java:123]: Please change the IP to `127.0.0.1`, and not use a real username.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sink/greenplum/GreenplumJdbcUtils.java [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+    public static List<GreenplumColumnInfo> getColumns(final Connection conn, final String schemaName,
+            final String tableName) throws Exception {
+        final List<GreenplumColumnInfo> columnList = new ArrayList<>();
+        final String querySql = GreenplumSqlBuilder.buildDescTableSql(schemaName, tableName);
+
+        try (Statement stmt = conn.createStatement();
+                ResultSet rs = stmt.executeQuery(querySql)) {
+            while (rs.next()) {
+                columnList.add(
+                        new GreenplumColumnInfo(
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sink/greenplum/GreenplumJdbcUtils.java:242]: Can those parameters be merged into one line?

Comment (haibo-duan) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sink/greenplum/GreenplumJdbcUtils.java:242]: OK

**Review by leosanqing — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Review by healchow — APPROVED** [review]

**Pull request merged**